shacl12-core/index.html

```diff
@@ -6290,6 +6289,170 @@ <h3>sh:group</h3>
 						</tr>
 					</table>
 				</div>
+				<p>
+					Note that the same information would be generated by the following example,
+					which changes the order of the triples but not the <code>sh:order</code> values:
+				</p>
+				<aside class="example">
+					<div class="shapes-graph">
+						<div class="turtle">
+ex:PersonFormShape
+	a sh:NodeShape ;
+	sh:property [
+		sh:path ex:lastName ;
+		sh:name "last name" ;
+		sh:description "The person's last name" ;
+		sh:order 1 ;
+		sh:group ex:NameGroup ;
+	] ;
+	sh:property [
+		sh:path ex:firstName ;
+		sh:name "first name" ;
+		sh:description "The person's given name(s)" ;
+		sh:order 0 ;
+		sh:group ex:NameGroup ;
+	] ;
+	sh:property [
+		sh:path ex:postalCode ;
+		sh:name "postal code" ;
+		sh:name "zip code"@en-us ;
+		sh:description "The postal code of the locality" ;
+		sh:order 13 ;
+		sh:group ex:AddressGroup ;
+	] ;
+	sh:property [
+		sh:path ex:streetAddress ;
+		sh:name "street address" ;
+		sh:description "The street address including number" ;
+		sh:order 11 ;
+		sh:group ex:AddressGroup ;
+	] ;
+	sh:property [
+		sh:path ex:locality ;
+		sh:name "locality" ;
+		sh:description "The suburb, city or town of the address" ;
+		sh:order 12 ;
+		sh:group ex:AddressGroup ;
+	] .
+	
+ex:AddressGroup
+	a sh:PropertyGroup ;
+	sh:order 1 ;
+	rdfs:label "Address" .
+
+ex:NameGroup
+	a sh:PropertyGroup ;
+	sh:order 0 ;
+	rdfs:label "Name" .
+						</div>
+						<div class="jsonld">
+							<pre class="jsonld">{
+	"@graph": [
+		{
+			"@id": "ex:PersonFormShape",
+			"@type": "sh:NodeShape",
+			"sh:property": [
+				{
+					"sh:description": "The person's last name",
+					"sh:group": {
+						"@id": "ex:NameGroup"
+					},
+					"sh:name": "last name",
+					"sh:order": {
+						"@type": "xsd:integer",
+						"@value": "1"
+					},
+					"sh:path": {
+						"@id": "ex:lastName"
+					}
+				},
+				{
+					"sh:description": "The person's given name(s)",
+					"sh:group": {
+						"@id": "ex:NameGroup"
+					},
+					"sh:name": "first name",
+					"sh:order": {
+						"@type": "xsd:integer",
+						"@value": "0"
+					},
+					"sh:path": {
+						"@id": "ex:firstName"
+					}
+				},
+				{
+					"sh:description": "The postal code of the locality",
+					"sh:group": {
+						"@id": "ex:AddressGroup"
+					},
+					"sh:name": [
+						"postal code",
+						{
+							"@language": "en-us",
+							"@value": "zip code"
+						}
+					],
+					"sh:order": {
+						"@type": "xsd:integer",
+						"@value": "13"
+					},
+					"sh:path": {
+						"@id": "ex:postalCode"
+					}
+				},
+				{
+					"sh:description": "The street address including number",
+					"sh:group": {
+						"@id": "ex:AddressGroup"
+					},
+					"sh:name": "street address",
+					"sh:order": {
+						"@type": "xsd:integer",
+						"@value": "11"
+					},
+					"sh:path": {
+						"@id": "ex:streetAddress"
+					}
+				},
+				{
+					"sh:description": "The suburb, city or town of the address",
```

Comment:
This should perhaps lose the "suburb" since that is generally an adjective describing towns around a city (the "urb").
```suggestion
					"sh:description": "The suburb, city, or town of the address",
```